frontend/src/components/partners/messages.js

```diff
@@ -0,0 +1,154 @@
+import { defineMessages } from 'react-intl';
+
+/**
+ * Internationalized messages for use on teams and orgs.
+ */
+export default defineMessages({
+  notAllowedPartners: {
+    id: 'management.messages.notAllowedPartners',
+    defaultMessage: 'You are not allowed to manage partners.',
+  },
+  managers: {
+    id: 'management.fields.managers',
+    defaultMessage: 'Managers',
+  },
+  remove: { id: 'management.fields.managers.remove', defaultMessage: 'Remove' },
+  add: { id: 'management.fields.managers.add', defaultMessage: 'Add' },
+  link: { id: 'management.fields.managers.link', defaultMessage: 'Link' },
+  noManagers: {
+    id: 'management.fields.managers.empty',
+    defaultMessage: 'There are no managers yet.',
+  },
+  startMapping: {
+    id: 'management.partnersCurrentProjects.startMapping',
+    defaultMessage: 'Start mapping',
+  },
+  manage: {
+    id: 'management.link.manage',
+    defaultMessage: 'Manage {entity}',
+  },
+  buildingsStats: {
+    id: 'management.filter.stats.buildingsStats',
+    defaultMessage: 'Building Edits',
+  },
+  contributors: {
+    id: 'management.contributors',
+    defaultMessage: 'Contributors',
+  },
+  roadsStats: {
+    id: 'management.filter.stats.roadsStats',
+    defaultMessage: 'km of Roads',
+  },
+  editsStats: {
+    id: 'management.filter.stats.editsStats',
+    defaultMessage: 'Total Edits',
+  },
+  communityStats: {
+    id: 'management.filter.stats.communityStats',
+    defaultMessage: 'Community stats',
+  },
+  mappersStats: {
+    id: 'management.filter.stats.mappersStats',
+    defaultMessage: 'Contributors',
+  },
+  noProjectContributors: {
+    id: 'project.detail.contributorCount.zero',
+    defaultMessage: 'No contributors yet',
+  },
+  edits: {
+    id: 'management.partnersActivity.edits',
+    defaultMessage: 'Edits',
+  },
+  buildings: {
+    id: 'management.partnersActivity.buildings',
+    defaultMessage: 'Buildings Added',
+  },
+  roads: {
+    id: 'management.partnersActivity.roads',
+    defaultMessage: 'km of Roads Added',
+  },
+  users: {
+    id: 'management.partnersActivity.users',
+    defaultMessage: 'Users',
+  },
+  readMore: {
+    id: 'project.readMoreButton',
```

Comment:
This message id is duplicated, it was already defined in `components/projectDetail/messages.js`. 

All ids in this file should start with the prefix `management.partner` to prevent conflicts.